src/components/LinkCard.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { Icon } from "astro-icon";
+import { classNames } from "../utils/classNames";
+
+type Props = {
+  name: string;
+  text: string;
+  href: string;
+};
+
+const { name, text, href } = Astro.props;
+---
+
+<a
+  href={href}
+  target="_blank"

```

Comment:
When using `target="_blank"` you should always set `rel="noopener noreferrer"` (https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever)